Sources/SwiftKafka/Utilities/BlockingCall.swift

```diff
@@ -0,0 +1,10 @@
+import Dispatch
+
+// performs blocking calls outside of cooperative thread pool
+internal func performBlockingCall<T>(queue: DispatchQueue, body: @escaping () -> T) async -> T {
+    await withCheckedContinuation { continuation in
```

Comment:
I am not sure that there is a good way to use swift concurrency with blocking calls.
The only thing is to use some separate thread to avoid contract violation